sdk/ml/azure-ai-ml/azure/ai/ml/constants/_compute.py

```diff
@@ -31,3 +31,11 @@ class ComputeDefaults:
     MAX_NODES = 4
     IDLE_TIME = 1800
     PRIORITY = "Dedicated"
+
+class CustomApplicationDefaults:
+    TARGET_PORT = "target_port"
+    PUBLISHED_PORT = "published_port"
+    TARGET_PORT_MIN_VALUE = 1
+    TARGET_PORT_MAX_VALUE = 65535
+    PUBLISHED_PORT_MIN_VALUE = 1025
```

Comment:
Where do these min/max values come from?

Reply:
1 to 65535 is the range of possible SSH ports. Target ports are free to be any of these.
1025 to 65535 is the range of unprivileged SSH ports. Published ports are limited to unprivileged ports.